js/admin.block-post.js

```diff
@@ -13,6 +13,7 @@
 	$(function() {
 		blockTitleInit();
 		blockIconInit();
+		blockCustomCategoryIconInit();
```

Comment:
I think we can entirely remove icons from our custom categories.

Reply:
Roger that.